repo/content/content_upgrade_lock.go

```diff
@@ -122,11 +127,42 @@ func (l *UpgradeLock) totalDrainInterval() time.Duration {
 	return l.MaxPermittedClockDrift + 2*l.IODrainTimeout
 }
 
+func (l *UpgradeLock) checkCoordinatorLock(ctx context.Context) (bool, error) {
+	req, err := http.NewRequestWithContext(ctx, "GET", l.CoordinatorURL, http.NoBody)
```

Comment:
would it make sense to send some identifying information here (say hash of username@hostname) so that the server knows who's connecting?

Reply:
will do

Reply:
used owner-id and process-id for now, see latest commit